patches/server/0005-Backport-PlayerHandshakeEvent-from-Paper.patch

```diff
@@ -20,8 +20,7 @@ index 215648ece51b9e0316bae17de60abccfd40f8b35..b7615b1472470805e6ee2a49dd8d4cdc
 +                if (com.destroystokyo.paper.event.player.PlayerHandshakeEvent.getHandlerList().getRegisteredListeners().length != 0) { // Hello? Can you hear me?
 +                    java.net.SocketAddress socketAddress = this.b.l;
 +                    String hostnameOfRemote = socketAddress instanceof java.net.InetSocketAddress ? ((java.net.InetSocketAddress) socketAddress).getHostString() : InetAddress.getLoopbackAddress().getHostAddress();
-+                    com.destroystokyo.paper.event.player.PlayerHandshakeEvent event = new com.destroystokyo.paper.event.player.PlayerHandshakeEvent(packethandshakinginsetprotocol.hostname, hostnameOfRemote, !proxyLogicEnabled);
-+                    org.bukkit.Bukkit.getPluginManager().callEvent(event);
++                    com.destroystokyo.paper.event.player.PlayerHandshakeEvent event = org.bukkit.craftbukkit.event.CraftEventFactory.callEvent(new com.destroystokyo.paper.event.player.PlayerHandshakeEvent(packethandshakinginsetprotocol.hostname, hostnameOfRemote, !proxyLogicEnabled));
```

Comment:
Any reason to use CraftEventFactory instead of "patches/api/0025-Add-a-call-helper-to-Event.patch" like you did in SpawnerPreSpawnEvent?